lightning/src/blinded_path/payment.rs

```diff
@@ -119,6 +119,22 @@ impl Writeable for ReceiveTlvs {
 	}
 }
 
+// This will be removed once we support forwarding blinded HTLCs, because we'll always read a
+// `BlindedPaymentTlvs` instead.
+impl Readable for ReceiveTlvs {
+	fn read<R: io::Read>(r: &mut R) -> Result<Self, DecodeError> {
+		_init_and_read_tlv_stream!(r, {
+			(1, _padding, option),
```

Comment:
Do we need to explicitly state the type?

Reply:
I think so although this will never be written at the moment anyway. Really dislike that ser macro quirk :( Removed the field for now.